lib/parsers/file-parser.ts

```diff
@@ -0,0 +1,20 @@
+import { ParseError } from '../errors';
+import { ComposerJsonFile, ComposerLockFile } from '../types';
+
+export class FileParser {
+    public static parseLockFile(lockFileContent: string): ComposerLockFile {
+        try {
+            return JSON.parse(lockFileContent);
+        } catch (e) {
+            throw new ParseError('Failed to parse lock file');
```

Comment:
the original error gets swallowed and never printed or logged anywhere? That will cause problems with debugging.
I'd include the original error message in the ParseError.